From 0c522e5cedefec055db5dcb1872d9a52ae94bebc Mon Sep 17 00:00:00 2001
From: Morris Hafner <hafnermorris@gmail.com>
Date: Tue, 7 May 2019 23:33:51 +0100
Subject: [PATCH] Set CMAKE_C_STANDARD

---
 CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 580d264..43a03ec 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,8 +1,11 @@
-cmake_minimum_required (VERSION 2.8.11)
+cmake_minimum_required (VERSION 3.1)
 
 project (OPENCL_ICD_LOADER)
 find_package (Threads REQUIRED)
 
+set(CMAKE_C_STANDARD 99)
+set(CMAKE_C_STANDARD_REQUIRED ON)
+
 # The option below allows building the ICD Loader library as a shared library
 # (ON, default) or a static library (OFF).
 #
-- 
2.21.0

